NEWS.md

```diff
@@ -38,6 +38,8 @@
   #96, fix: #111, @zkamvar)
 * Documents with no trailing newline will no longer throw a warning (issue: #65;
   fix: #114, @zkamvar)
+* Documents with dollar signs but no math will no longer fail with the
```

Comment:
Hey I really like your tracking both issue and fix!

Reply:
Thank you!